docs/content/reference/03-middleware.md

```diff
@@ -336,6 +336,216 @@ module.exports = {
 }
 ```
 
+### `validate.body`
```

Comment:
These three sections are going to read _very_ similarly. The key things to look for are:
- Whether the middleware prevents the handler from being executed if the schema validation fails (query and param block execution, while body does not.)
- The default settings used for `Ajv`. (query and param coerce types because the source data is guaranteed to be a string. body params are first interpreted by the body processing stack and may represent other value types, thus `coerceTypes` is off.)
- The examples are tweaked for each type.